devnet/src/types.rs

```diff
@@ -95,7 +160,7 @@ pub struct ParsedConfig {
 
 pub async fn load_config() -> ParsedConfig {
     const CONFIG_FILE: &str = "config.yaml";
-    let config = std::fs::read_to_string(CONFIG_FILE).unwrap();
+    let config = std::fs::read_to_string(CONFIG_FILE).expect("Did not find 'config.yaml'");
```

Comment:
Thanks a lot for adding `expect` and an explanation to it.

Nit:
expect statements should contain the reason you _expect_ the Result to be Ok, not why it became an error.
I.e. something like 
```suggestion
    let config = std::fs::read_to_string(CONFIG_FILE).expect("A `config.yaml` should exist in the working directory.");
```

Here's a nice link from the std docs for `expect()`:
https://doc.rust-lang.org/std/result/enum.Result.html#recommended-message-style